### PR TITLE
BBCode: add new buttons where it exists

### PR DESCRIPTION
**Pull request by MegaApplePi**
## Description

add new buttons

## Status

Done

## Changelog

- remove code language
- update and rename forum post editor
- add suffix "-old" to old buttons
- use paragraph not single lines

see commits for future updates

## Related Issues

~~conflicts #792~~

### CODE REVIEW

**Comment by TPGPL** [comment]
i think you shouldnt keep the old buttons tbh

**Comment by MegaApplePi** [comment]
alright, removed

File: wiki/BBCode/en.md [hunk 1]
```diff
@@ -1,74 +1,67 @@
...
+### BBCode button behaviour
 
 Without any text highlighted, clicking the button will surround the tags around the text cursor placement in the post editor.
 
 With text highlighted in the post editor, clicking the button will surround the tags around the highlighted text.
 
 ### Combining tags
 
-Tags can be combined for fancier/richer formatting.
+Tags can be combined for fancier and/or richer formatting.
```
Comment (TPGPL) [inline wiki/BBCode/en.md]: delete `/or` i think?

File: wiki/BBCode/en.md [hunk 2]
```diff
@@ -1,74 +1,67 @@
...
 ### Underline
 
 **Draws a horizontal line underneath** the word(s) or an entire paragraph.
 
-- Dedicated button: ![Underline button](img/underline.jpg "Underline button")
+- Dedicated button: N/A
 - Notes:
-  - Use sparingly.
-    Overuse may create discomfort and making the post harder to read.
+  - Use sparingly. Overuse may create discomfort and making the post harder to read.
```
Comment (TPGPL) [inline wiki/BBCode/en.md]: and make the post (same in the others)

File: wiki/BBCode/en.md [hunk 3]
```diff
@@ -338,48 +327,45 @@ Where the bullets are placed inside the wrapper.
...
-Using `C:\Users\Name\Pictures\image.jpg` will **not** work!
+**Do not link the image directly from a local filepath!** Using `C:\Users\Name\Pictures\image.jpg` will **not work**.
 
-Please upload the image to reputable image sharing sites like [imgur](https://imgur.com/ "imgur") or [puush](https://puush.me/ "puush").
-Once the image is uploaded successfully, copy and paste the direct link provided in-between the image tags.
+Please upload the image to reputable image sharing sites like [imgur](https://imgur.com/ "imgur") or [puush](https://puush.me/ "puush"). Once the image is uploaded successfully, copy and paste the direct link provided in-between the image tags.
 
-Also note, some websites don't appreciate direct links to their images (otherwise known as _hotlinks_).
-Image sharing sites, like the ones listed above, are most likely okay with direct links since they are image sharing sites.
+Also note, some websites don't appreciate direct links to their images (otherwise known as _hotlinks_). Image sharing sites, like the ones listed above, are most likely okay with direct links since they are image sharing sites.
```
Comment (TPGPL) [inline wiki/BBCode/en.md]: Also note that

File: wiki/BBCode/en.md [hunk 4]
```diff
@@ -388,35 +374,35 @@ This is usually used to denote a new section in your post.
...
+```
 [notice]
 ...
 [/notice]
 ```
 
 ## Trivia
 
-- Original forum post: [HOW TO: Forum BBCodes](https://osu.ppy.sh/forum/t/445599) by [Stefan](https://osu.ppy.sh/u/626907)
+- Original forum post: [HOW TO: Forum BBCodes](/community/forums/topics/445599) by [Stefan](/users/626907)
```
Comment (TPGPL) [inline wiki/BBCode/en.md]: why not full links

**Pull request merged**